charts/golang/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: golang
 description: A chart for Golang.
 icon: https://golang.org/lib/godoc/images/go-logo-blue.svg
-version: 5.3.0
-appVersion: 5.3.0
+version: 5.4.0
+appVersion: 5.4.0
```

Comment:
Because we're changing the API for the postgres inputs, let's bump to `6.0.0` as it is a breaking change.

Reply:
Good point.  Done!